crates/node/src/lib.rs

```diff
@@ -379,12 +379,9 @@ impl Node {
 
         #[cfg(feature = "replicated-loglet")]
         if let Some(log_server) = self.log_server {
-            tc.spawn(
-                TaskKind::SystemBoot,
-                "log-server-init",
-                None,
-                log_server.start(metadata_writer),
-            )?;
+            log_server
+                .start(metadata_writer, &mut self.server_builder)
+                .await?;
         }
```

Comment:
I think this will conflict with https://github.com/restatedev/restate/pull/2185 where I needed to move the `node-rpc-server` task to the front of the `start` method.

Reply:
Passing the `server_builder` into the `LogServerService::create` method should work, though.

Reply:
Maybe it also fine to move the log-server start to the top of the `start` method.

Reply:
Can't move it to `create()` because we only create the logstore at start() time.